### PR TITLE
[11.0][MIG] quality_control

### PR DESCRIPTION
**Pull request by ProcessControl-JHF**
cc @guadaltech

### CODE REVIEW

File: quality_control/models/qc_inspection.py [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+
+
+class QcInspection(models.Model):
+    _name = 'qc.inspection'
+    _description = 'Quality control inspection'
+    _inherit = ['mail.thread', 'mail.activity.mixin']
+
+    @api.depends('inspection_lines', 'inspection_lines.success')
+    def _compute_success(self):
+        self.success = all([x.success for x in self.inspection_lines])
```
Comment (LoisRForgeFlow) [inline quality_control/models/qc_inspection.py]: adapt for recordsets

File: quality_control/models/qc_inspection.py [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+
+    @api.multi
+    def _links_get(self):
+        link_obj = self.env['res.request.link']
+        return [(r.object, r.name) for r in link_obj.search([])]
+
+    @api.depends('object_id')
+    def _compute_product_id(self):
+        if self.object_id and self.object_id._name == 'product.product':
```
Comment (LoisRForgeFlow) [inline quality_control/models/qc_inspection.py]: same

**Review by LoisRForgeFlow — COMMENTED** [review]
I just wanted to recall two old api.one methods that weren't still adapted to handle recordsets. You fixed most of them but I believe you missed these two.

Thanks for the migration.

File: quality_control/models/qc_inspection.py (hunk 1 above)
Comment (ProcessControl-JHF) [reply quality_control/models/qc_inspection.py]: done, thanks

File: quality_control/models/qc_inspection.py (hunk 2 above)
Comment (ProcessControl-JHF) [reply quality_control/models/qc_inspection.py]: done, thanks

**Review by LoisRForgeFlow — APPROVED** [review]
Thanks!

**Review by pedrobaeza — APPROVED** [review]

**Comment by SalahAdDin** [comment]
@jhumfer 
I think i found a problem:
noestalogo
It is the same in the dashboard, but i don't know why.

**Comment by ProcessControl-JHF** [comment]
@SalahAdDin If you add the text _quality_control,static/description/icon.png_ to the field _Web icon File_ from the _Quality Control_ menu, the image appears in the dashboard

**Comment by SalahAdDin** [comment]
But it must be done here in the module because other modules are working correctly.

**Comment by pedrobaeza** [comment]
Please propose any of you the improvement. The module should adapt also to new design guidelines, but that's another history.

**Comment by SalahAdDin** [comment]
@jhumfer Yes, you are right,
@pedrobaeza Should i make a small pull request for this?

**Comment by pedrobaeza** [comment]
Please

**Comment by SalahAdDin** [comment]
https://github.com/OCA/manufacture/pull/304

**Pull request merged**